File: SpaceInvaders/GameObject/Alien/JellyFishAlien.cs
using System;
using System.Diagnostics;

namespace SpaceInvaders
{
    class JellyfishAlien : Alien
    {
        int jellyfishPoints = 10;

        public JellyfishAlien(GameObject.Name name, Sprite.Name spriteName, float posX, float posY)
            : base(name, spriteName)
        {
            this.x = posX;
            this.y = posY;
        }

        ~JellyfishAlien()
        {

        }

        public override int GetPoints()
        {
            return jellyfishPoints;
        }

        public override void Update()
        {
            base.Update();
        }

    }
}

File: quarkus-migration/space-invaders-quarkus/src/main/java/net/beeland/spaceinvaders/gameobject/JellyfishAlien.java
package net.beeland.spaceinvaders.gameobject;

/**
 * JellyfishAlien - Bottom rows alien, worth 10 points
 */
public class JellyfishAlien extends Alien {
    
    private static final int JELLYFISH_POINTS = 10;
    
    /**
     * Constructor
     *
     * @param posX Initial X position
     * @param posY Initial Y position
     */
    public JellyfishAlien(float posX, float posY) {
        super(GameObject.GameObjectName.JELLYFISH_ALIEN);
        this.x = posX;
        this.y = posY;
    }
    
    @Override
    public int getPoints() {
        return JELLYFISH_POINTS;
    }
    
    @Override
    public Type getType() {
        return Type.JELLYFISH;
    }
    
    @Override
    public void update() {
        super.update();
    }
}